src/mixins/map-mixin.js

```diff
@@ -277,8 +277,9 @@ export default function mapMixin(
     mapboxlogo.target = "_blank"
     mapboxlogo.innerHTML = "Mapbox"
 
-    const existingLogo = document.getElementsByClassName('mapbox-maplogo')
-    if(existingLogo.length  ) {
+    const thisMapContainer = _map._container
```

Comment:
How certain are we `_map` will never still be null here?

Reply:
Added check for null `_map`